[2022-07-05 07:21:34,056] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [queued]>
[2022-07-05 07:21:34,111] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [queued]>
[2022-07-05 07:21:34,112] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:34,113] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:21:34,113] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:34,178] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 23:00:00+00:00
[2022-07-05 07:21:34,202] {standard_task_runner.py:52} INFO - Started process 596 to run task
[2022-07-05 07:21:34,218] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T23:00:00+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp3xsb_ssk', '--error-file', '/tmp/tmp87_6yg0j']
[2022-07-05 07:21:34,221] {standard_task_runner.py:77} INFO - Job 252: Subtask kimuksung_test
[2022-07-05 07:21:34,467] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:21:34,736] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T23:00:00+00:00
[2022-07-05 07:21:34,739] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:21:34,739] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:34,809] {taskinstance.py:1282} INFO - Marking task as FAILED. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T230000, start_date=20220705T072134, end_date=20220705T072134
[2022-07-05 07:21:34,856] {standard_task_runner.py:92} ERROR - Failed to execute job 252 for task kimuksung_test
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:34,898] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-07-05 07:21:35,145] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 07:25:45,723] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [queued]>
[2022-07-05 07:25:45,778] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [queued]>
[2022-07-05 07:25:45,778] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:45,778] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:25:45,778] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:45,856] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 23:00:00+00:00
[2022-07-05 07:25:45,871] {standard_task_runner.py:52} INFO - Started process 786 to run task
[2022-07-05 07:25:45,891] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T23:00:00+00:00', '--job-id', '283', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpn7_kwjm6', '--error-file', '/tmp/tmps015z6wl']
[2022-07-05 07:25:45,892] {standard_task_runner.py:77} INFO - Job 283: Subtask kimuksung_test
[2022-07-05 07:25:46,139] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:25:46,614] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T23:00:00+00:00
[2022-07-05 07:25:46,617] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:25:46,620] {test.py:22} INFO - pubg
[2022-07-05 07:25:46,621] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 07:25:46,718] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T230000, start_date=20220705T072545, end_date=20220705T072546
[2022-07-05 07:25:46,780] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 07:25:46,883] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:06:21,001] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [queued]>
[2022-07-05 08:06:21,128] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [queued]>
[2022-07-05 08:06:21,129] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:21,129] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:06:21,129] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:21,408] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 23:00:00+00:00
[2022-07-05 08:06:21,416] {standard_task_runner.py:52} INFO - Started process 2169 to run task
[2022-07-05 08:06:21,449] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T23:00:00+00:00', '--job-id', '335', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpqkc1x_76', '--error-file', '/tmp/tmpon769lfx']
[2022-07-05 08:06:21,450] {standard_task_runner.py:77} INFO - Job 335: Subtask kimuksung_test
[2022-07-05 08:06:22,433] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:06:23,121] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T23:00:00+00:00
[2022-07-05 08:06:23,122] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:06:23,123] {test.py:22} INFO - pubg
[2022-07-05 08:06:23,123] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 08:06:23,303] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T230000, start_date=20220705T080621, end_date=20220705T080623
[2022-07-05 08:06:23,503] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:06:24,163] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:11:30,571] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [queued]>
[2022-07-05 08:11:30,630] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [queued]>
[2022-07-05 08:11:30,631] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:30,632] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:11:30,632] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:30,734] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-04 23:00:00+00:00
[2022-07-05 08:11:30,749] {standard_task_runner.py:52} INFO - Started process 2468 to run task
[2022-07-05 08:11:30,762] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-04T23:00:00+00:00', '--job-id', '401', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpws47_pgj', '--error-file', '/tmp/tmpfbbxb6yf']
[2022-07-05 08:11:30,763] {standard_task_runner.py:77} INFO - Job 401: Subtask kimuksung_test
[2022-07-05 08:11:31,012] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-04T23:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:11:31,182] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T23:00:00+00:00
[2022-07-05 08:11:31,184] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:11:31,184] {test.py:22} INFO - pubg
[2022-07-05 08:11:31,185] {python.py:175} INFO - Done. Returned value was: pubg
[2022-07-05 08:11:31,242] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220704T230000, start_date=20220705T081130, end_date=20220705T081131
[2022-07-05 08:11:31,284] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:11:31,371] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
